.tekton/run-tests-task.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: run-tests
```

Comment:
@maknop does this task need to be a separate task? Or can we define it in the pull/push pipelines that already exist?

Reply:
This task is for your integration test pipeline. It isn't apart of your push/pull pipelines. Once those pass, the integration test pipeline gets kicked off. The reason that task is in this repo is so you can cater it to your specific needs for this app. You also get the benefit of making changes to the task and being able to merge them yourself without us needing to merge them in bonfire-tekton. :)